### DISCUSSION TITLE
LibreSBC installation support

### DISCUSSION THREAD

**Discussion by DAOADAMA** [hnimminh/libresbc#128] [General]
Dear,
we need support for LibreSBC installation .
can someone provide paying support?
Thanks

**Comment by hnimminh** [comment]
Hi @DAOADAMA, 
There is installation guide in [wiki page](https://github.com/hnimminh/libresbc/wiki/Installation)  that you can use ansible install libresbc.
Feel free to email me if you till need a help on that.

**Comment by DAOADAMA** [comment]
Dear Minh,
Thanks you for your reply.
We woulike to deploy LibreSBC in production environnement so we need support in cas of failure.
We have tryed but ANSIBLE is not working fine.
Please advice
Br,


De : Minh ***@***.***>
Envoyé : mardi 10 octobre 2023 06:34
À : hnimminh/libresbc ***@***.***>
Cc : DAO ADAMA ***@***.***>; Mention ***@***.***>
Objet : Re: [hnimminh/libresbc] LibreSBC installation support (Discussion #128)


Hi @DAOADAMA<https://github.com/DAOADAMA>,
There is installation in guide in wiki page<https://github.com/hnimminh/libresbc/wiki/Installation> that you can use ansible install libresbc.
feel free to email me if you till need a help on that.
